com.unity.formats.fbx/Editor/ExportModelEditorWindow.cs

```diff
@@ -101,6 +101,122 @@ protected float FbxExtLabelWidth
 
         protected abstract ExportOptionsSettingsSerializeBase SettingsObject { get; }
 
+        // Helper functions for persisting the Export Settings for the session
+        protected abstract string SessionStoragePrefix { get; }
+
+        protected const string k_SessionSettingsName = "Settings";
+        protected const string k_SessionFbxPathsName = "FbxSavePath";
+        protected const string k_SessionSelectedFbxPathName = "SelectedFbxPath";
+        protected const string k_SessionPrefabPathsName = "PrefabSavePath";
+        protected const string k_SessionSelectedPrefabPathName = "SelectedPrefabPath";
+
+        protected void StorePathsInSession(string varName, List<string> paths)
+        {
+            if(paths == null)
+            {
+                return;
+            }
+
+            var n = paths.Count;
+            SessionState.SetInt(string.Format(SessionStoragePrefix, varName), n);
+            for (int i = 0; i < n; i++)
+            {
+                SessionState.SetString(string.Format(SessionStoragePrefix + "_{1}", varName, i), paths[i]);
+            }
+        }
+
+        protected void RestorePathsFromSession(string varName, List<string> defaultsPaths, out List<string> paths)
+        {
+            var n = SessionState.GetInt(string.Format(SessionStoragePrefix, varName), 0);
+            paths = new List<string>();
+            if (n > 0)
+            {
+                for (int i = 0; i < n; i++)
+                {
+                    var path = SessionState.GetString(string.Format(SessionStoragePrefix + "_{1}", varName, i), null);
+                    if (!string.IsNullOrEmpty(path))
+                    {
+                        paths.Add(path);
+                    }
+                }
+            }
+
+            if (paths.Count <= 0)
+            {
+                paths = defaultsPaths;
```

Comment:
This if can go to the top of the function, check if n == 0 then paths = defaultPaths, else paths = new List<string>